libbeat/_meta/config.full.yml

```diff
@@ -64,15 +64,18 @@
 #           http.code: 200
 #
 # The following example enriches each event with metadata from the cloud
-# provider about the host machine. It works on EC2, GCE, and DigitalOcean.
+# provider about the host machine. It works on EC2, GCE, DigitalOcean,
+# Tencent Cloud, and Alibaba Cloud.
 #
 #processors:
-#- add_cloud_metadata:
+#- add_cloud_metadata: ~
```

Comment:
It's a [null](http://yaml.org/spec/history/2002-04-07.html#trans-null) in YAML. It's not required but we wanted to have some kind of value on the RHS.